### PR TITLE
A few typos + function cleanup

### PR DESCRIPTION
**Pull request by donavon**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -727,22 +727,13 @@ function isObject(a) {
...
-    return (
-      document.visibilityState === 'visible' ||
-      document.visibilityState === 'prerender'
-    )
-  }
-
-  return true
+  return document.visibilityState === undefined
+    || document.visibilityState === 'visible'
+    || document.visibilityState === 'prerender'
```
Comment (tannerlinsley) [inline src/index.js]: The changes here are not logically equivalent

File: src/index.js [hunk 2]
```diff
@@ -727,22 +727,13 @@ function isObject(a) {
...
 }
 
 function isOnline() {
-  if (typeof navigator.onLine !== 'undefined') {
-    return navigator.onLine
-  }
-
-  return true
+  return navigator.onLine === undefined || navigator.onLine
```
Comment (tannerlinsley) [inline src/index.js]: I believe here too

File: src/index.js (hunk 1 above)
Comment (donavon) [reply src/index.js]: they are equivalent. see this [CodeSandbox](https://codesandbox.io/s/stupefied-http-l3v0p) console output

File: src/index.js (hunk 2 above)
Comment (donavon) [reply src/index.js]: also equivalent

File: src/index.js (hunk 1 above)
Comment (tannerlinsley) [reply src/index.js]: Ah. Right you are!

Comment (donavon) [reply src/index.js]: maybe some unit tests would be in order 😉

**Comment by tannerlinsley** [comment]
Indeed.

Tanner Linsley

**Comment by tannerlinsley** [comment]
Merged!

**Comment by tannerlinsley** [comment]
Released!

**Pull request merged**